broker/src/test/java/io/camunda/zeebe/broker/system/partitions/ZeebePartitionTest.java

```diff
@@ -366,8 +395,10 @@ public void shouldReportUnhealthyPerDefault() {
 
     final var zeebePartitionHealth = captor.getValue();
     final HealthReport healthReport = zeebePartitionHealth.getHealthReport();
+    zeebePartitionHealth.setServicesInstalled(true);
+
     assertThat(healthReport.getStatus()).isEqualTo(HealthStatus.UNHEALTHY);
-    assertThat(healthReport.getIssue().message()).contains("Initial state");
+    assertThat(healthReport.getIssue().message()).contains("Services not installed");
```

Comment:
:x: This is wrong: first we do `setServicesInstalled(true)` and then we assert that the health status is "Services not installed"?